src/test/java/seedu/address/model/StudyTrackerTest.java

```diff
@@ -38,18 +38,18 @@ public void resetData_null_throwsNullPointerException() {
 
     @Test
     public void resetData_withValidReadOnlyAddressBook_replacesData() {
```

Comment:
There are a few instances of AddressBook found in this class, should be changed to StudyTracker